tools/mergeLibs.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
```

Comment:
This whole file should follow the style of our other bash scripts; lowercase snake_case for functions/local vars, uppercase SNAKE_CASE for globals.